src/commands/qmod.rs

```diff
@@ -85,11 +87,33 @@ fn execute_qmod_create_operation(create_parameters: CreateQmodJsonOperationArgs)
         file_copies: Default::default(),
     };
 
-    json.write_template();
+    json.write(PathBuf::from(ModJson::get_template_name()));
 }
 
 fn execute_qmod_build_operation() {
-    // TODO: Make it work (probably executing cmake --build ./ or something like that)
+    if !std::path::Path::new("mod.template.json").exists() {
```

Comment:
Could you not remove TODOs that are not done? I don't think this actually builds yet

Reply:
it's not intended to build. A package manager shouldn't be responsible for running a command that a script should instead.
 In other words, make a script call `qpm qmod build` rather than have `qpm qmod build` run the script

Reply:
then put something like that there instead, instead of just removing that...